addon/components/detail.js

```diff
@@ -588,8 +592,14 @@ export default Component.extend(SpreadMixin, HookMixin, PropTypeMixin, {
         component.unregisterForFormValueChanges(component)
       })
 
+      const formValue = this.get('renderValue') || {}
+
       // Make sure we inform component of formValue immediately
-      component.formValueChanged(this.get('renderValue') || {})
+      if (typeOf(component.formValueChanged) === 'function') {
```

Comment:
is the northbound interface here guaranteed to either pass a `function` or a `task` here?